packages/angular-output-target/tsconfig.json

```diff
@@ -11,6 +11,7 @@
     "moduleResolution": "node",
     "module": "esnext",
     "target": "es2017",
+    "types": ["node"],
```

Comment:
These should be picked up automatically. By defining `types` here we restrict other types to be loaded as they are not defined in here.
```suggestion
```